src/Cli/ConfigGenerator.cs

```diff
@@ -991,20 +992,21 @@ public static bool TryStartEngineWithOptions(StartOptions options, FileSystemRun
             {
                 if (options.LogLevel is < LogLevel.Trace or > LogLevel.None)
                 {
-                    _logger.LogError($"LogLevel's valid range is 0 to 6, your value: {options.LogLevel}, see: " +
-                        $"https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=dotnet-plat-ext-7.0");
+                    _logger.LogError(
+                        "LogLevel's valid range is 0 to 6, your value: {logLevel}, see: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=dotnet-plat-ext-7.0",
```

Comment:
```suggestion
                        "LogLevel's valid range is 0 to 6, your value: {logLevel}, see: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=dotnet-plat-ext-6.0",
```
nit: .net6 and no locale in url